calendar_reboot.js

```diff
@@ -695,7 +694,9 @@ var LocalCalendarStorage = function(params) {
         storeInLocalStorage(toKey(calendarObj.state.uniqueId), calendarObj.state);
         
         //put calendar in allCalendarIdss and store it
+        var allCalendarIds = self.getAllCalendarIds();
```

Comment:
Nothing to change here, but in general this pattern is known as "read-modify-write" and it only works because we *know* that we are the only ones that are modifying the value.

If there were other code that could be modifying this, then you could end up "racing" where both of you read, then one of you modifies and writes but it gets "forgotten" when the second one writes because the version that the second one read didn't include the changes from the first.